### PR TITLE
fix(api): create in-app integrations when new org is created

### PR DESCRIPTION
**Pull request by jainpawan21**
### What changed? Why was the change needed?
In-app integrations should be created and active by default when user sign up and new org is created

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/api/src/app/integrations/usecases/create-novu-integrations/create-novu-integrations.usecase.ts [hunk 1]
```diff
@@ -89,8 +89,24 @@ export class CreateNovuIntegrations {
...
+        name: 'Novu In-App',
+        active: true,
+        check: false,
+        userId: command.userId,
+        environmentId: command.environmentId,
+        organizationId: command.organizationId,
+      })
+    );
+  }
+
```
Comment (ainouzgali) [inline apps/api/src/app/integrations/usecases/create-novu-integrations/create-novu-integrations.usecase.ts]: Let's also add a count check to be extra sure there is no existing in app integration (only one is allowed per environment)

**Review by ainouzgali — COMMENTED** [review]
@jainpawan21 left you a comment :) 
Please also run the web tests, to see none are failing there. Maybe there is a test that adds an in-app (which they won't be able to do anymore)

File: apps/api/src/app/integrations/usecases/create-novu-integrations/create-novu-integrations.usecase.ts (hunk 1 above)
Comment (SokratisVidros) [reply apps/api/src/app/integrations/usecases/create-novu-integrations/create-novu-integrations.usecase.ts]: Good call Gali, we need to ensure one in-app integration.

**Review by SokratisVidros — APPROVED** [review]

**Review by ainouzgali — APPROVED** [review]
🚀
